contracts/EverlongStrategy.sol

```diff
@@ -259,6 +311,90 @@ contract EverlongStrategy is BaseStrategy {
         return hasMaturedPositions() || canOpenPosition();
     }
 
+    /// @notice Sets the temporary tend configuration. Will only persist through
+    ///         the duration of the transaction. Must be called in the same tx
+    ///         as `tend()`.
+    function setTendConfig(
+        IEverlongStrategy.TendConfig memory _config
+    ) external onlyKeepers {
+        // HACK: Must hash these every time since only direct number constants
+        //       are supported by inline assembly.
```

Comment:
I think there is a workaround for this. The following failed to compile with `TypeError: Only direct number constants and references to such constants are supported by inline assembly.`:

```
// SPDX-License-Identifier: Apache-2.0
pragma solidity 0.8.24;

contract TStore {
    bytes32 public constant HASH = keccak256(abi.encode("KEY"));

    function tStore() external {
        assembly {
            tstore(HASH, 1)
        }
    }
}
```

whereas this version succeeds:

```
// SPDX-License-Identifier: Apache-2.0
pragma solidity 0.8.24;

contract TStore {
    bytes32 public constant HASH = keccak256(abi.encode("KEY"));

    function tStore() external {
        bytes32 hash = HASH;
        assembly {
            tstore(hash, 1)
        }
    }
}
```

Reply:
Appreciate you finding this! Will update accordingly